api/go.mod

```diff
@@ -5,8 +5,8 @@ go 1.18
 require (
 	github.com/fluxcd/pkg/apis/kustomize v0.4.2
```

Comment:
Increment this too to `v0.5.3` ?

Reply:
There is no such version, 0.4.2 is the latest one for the API package.

Reply:
oh, confused with the path :sweat_smile: